src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/BaseControllerTest.java

```diff
@@ -33,12 +33,24 @@ void doEmptyCaseDataSetUp() throws IOException, URISyntaxException {
                 .getResource("/fixtures/empty-casedata.json").toURI()));
     }
 
+    void setTestCaseDataFromResource(String pathToJson) throws IOException, URISyntaxException {
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Can we use a common instance of ObjectMapper?

Reply:
Ended up removing this method entirely